### PR TITLE
Removed custom focus styles, now use browser native focus

### PR DESCRIPTION
**Pull request by chrisvanmook**

### CODE REVIEW

File: packages/asc-ui/src/components/Button/ButtonStyle.ts [hunk 1]
```diff
@@ -2,7 +2,6 @@ import { darken, readableColor, size, transitions } from 'polished'
 import styled, { css } from 'styled-components'
```
Comment (jonkoops) [inline packages/asc-ui/src/components/Button/ButtonStyle.ts:2]: There are still two instances in this file where the `outline` is set which might interfere with the browser styling.

File: packages/asc-ui/src/components/FilterTag/FilterTagStyle.ts [hunk 2]
```diff
@@ -1,37 +1,29 @@
...
 
-    ${IconStyle} {
-      margin: 0 0 0 ${themeSpacing(2)};
-      vertical-align: middle;
-    }
-  `
-}
-
-export default styled(TypographyStyle)<Props>`
-  ${FilterTagStyleCSS}
```
Comment (jonkoops) [inline packages/asc-ui/src/components/FilterTag/FilterTagStyle.ts:36]: I have no idea why this was like this, but good that you re-factored it :+1:

File: packages/asc-ui/src/components/Checkbox/CheckboxStyle.ts [hunk 3]
```diff
@@ -1,6 +1,5 @@
 import styled, { css } from 'styled-components'
```
Comment (jonkoops) [inline packages/asc-ui/src/components/Checkbox/CheckboxStyle.ts:1]: There is still some styling present that modifies the `outline` in this file.

File: packages/asc-ui/src/components/ShowMoreShowLess/ShowMoreShowLessStyle.ts [hunk 4]
```diff
@@ -5,6 +5,7 @@ export type ShowMoreShowLessProps = {
   maxHeight?: string
   showMoreText?: string
   showLessText?: string
+  gradientBackground?: string
```
Comment (jonkoops) [inline packages/asc-ui/src/components/ShowMoreShowLess/ShowMoreShowLessStyle.ts:8]: This seems an unrelated change, I think it should be removed.

File: packages/asc-ui/src/index.ts [hunk 5]
```diff
@@ -115,12 +115,6 @@ export {
   svgFill,
   themeSpacing,
 } from './utils'
-export {
-  fillWhenFocused,
-  focusFill,
-  focusOutline,
-  outlineWhenFocused,
-} from './utils/focus'
```
Comment (jonkoops) [inline packages/asc-ui/src/index.ts:123]: :fire: :fire: :fire:

File: stories/src/ui/ShowMoreShowLess.stories.mdx [hunk 6]
```diff
@@ -1,6 +1,6 @@
 import styled from 'styled-components'
```
Comment (jonkoops) [inline stories/src/ui/ShowMoreShowLess.stories.mdx:1]: Same for this change, seems unrelated.

File: packages/asc-ui/src/components/Button/ButtonStyle.ts (hunk 1 above)
Comment (chrisvanmook) [reply packages/asc-ui/src/components/Button/ButtonStyle.ts:2]: only for hover and disabled state. shouldn't interfere I guess, right?

File: packages/asc-ui/src/components/Checkbox/CheckboxStyle.ts (hunk 3 above)
Comment (chrisvanmook) [reply packages/asc-ui/src/components/Checkbox/CheckboxStyle.ts:1]: We need the outline rules here as well.

File: packages/asc-ui/src/components/ShowMoreShowLess/ShowMoreShowLessStyle.ts (hunk 4 above)
Comment (chrisvanmook) [reply packages/asc-ui/src/components/ShowMoreShowLess/ShowMoreShowLessStyle.ts:8]: Will open a separate PR for this. excuse my laziness 😉

File: packages/asc-ui/src/components/Button/ButtonStyle.ts (hunk 1 above)
Comment (jonkoops) [reply packages/asc-ui/src/components/Button/ButtonStyle.ts:2]: Yeah, should be ok. Just double checking.

File: packages/asc-ui/src/components/Checkbox/CheckboxStyle.ts (hunk 3 above)
Comment (jonkoops) [reply packages/asc-ui/src/components/Checkbox/CheckboxStyle.ts:1]: :+1:
